eth/main.cpp

```diff
@@ -866,7 +866,7 @@ int main(int argc, char** argv)
 				cout << "  with seed as " << seedHash << endl;
 				if (valid)
 					cout << "(mixHash = " << r.mixHash << ")" << endl;
-				cout << "SHA3( light(seed) ) = " << sha3(EthashAux::light((uint64_t)bi.number)->data()) << endl;
+				cout << "SHA3( light(seed) ) = " << sha3(EthashAux::light(bi)->data()) << endl;
```

Comment:
line length isn't everything :)

by passing BlockInfo rather than seedhash, more data is being passed yet merely the same amount of information. this makes possible several errors (e.g. trying to use bi.number) that wouldn't otherwise be possible.

even though you know you're not going to make that mistake right now, it's generally better to organise the structure of code to make such things impossible.